backend/Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN mkdir lastfm_images
 # code doesn't require these steps to run again
 COPY requirements.txt .
 COPY setup.py .
-COPY initDB.py .
+COPY init_db.py .
```

Comment:
real small, but i actually dont use this file anymore if you dont mind removing it in this pr (or another one lol)